Client/Frontend/Home/FxHomeJumpBackInViewModel.swift

```diff
@@ -41,41 +45,67 @@ class FirefoxHomeJumpBackInViewModel: FeatureFlagsProtocol {
         self.tabManager = tabManager
     }
 
-    var layoutVariables: JumpBackInLayoutVariables {
-        let horizontalVariables = JumpBackInLayoutVariables(columns: 2, scrollDirection: .horizontal, maxItemsToDisplay: 4)
-        let verticalVariables = JumpBackInLayoutVariables(columns: 1, scrollDirection: .vertical, maxItemsToDisplay: 2)
+    // The maximum number of items to display in the whole section
+    static var maxItemsToDisplay: Int {
+        if deviceIsiPad {
```

Comment:
nit: The if~else chain is fine but we can be further clean it (see below). 
Comments might be bit much as `deviceIsiPad` is very self explanatory. What do you think? @lmarceau 🤔 

```suggestion
return deviceIsiPad ? 3 : deviceIsInLandscapeMode ? 4 : 2
```